app/uk/gov/hmrc/teamsandrepositories/config/TeamsAndRepositoriesConfig.scala

```diff
@@ -38,21 +38,23 @@ class UrlTemplatesProvider @Inject()(configuration: Configuration) {
   val ciUrlTemplates: UrlTemplates = UrlTemplates(getTemplatesForEnvironments())
 
   private def urlTemplates =
-    configuration.getConfig("url-templates").getOrElse(throw new RuntimeException("no url-templates config found"))
+    configuration
+      .getOptional[Configuration]("url-templates")
+      .getOrElse(throw new RuntimeException("no url-templates config found"))
 
   private def getTemplatesForEnvironments(): ListMap[String, Seq[UrlTemplate]] = {
     val configs = urlTemplates
-      .getConfigSeq("envrionments")
+      .getOptional[Seq[Configuration]]("envrionments")
```

Comment:
doesn't `get[A](key)` already throw an exception with key in the message? Rather than `getOptional[A](key).getOrElse(sys.error(s"missing config $key")`
If not, maybe a helper for this pattern?